api/v1beta1/disruption_webhook_test.go

```diff
@@ -371,6 +371,44 @@ var _ = Describe("Disruption", func() {
 				})
 			})
 
+			When("triggers.*.notBefore is in the past", func() {
+				It("triggers.inject should return an error", func() {
+					newDisruption.Spec.Duration = "30m"
+					newDisruption.Spec.Triggers = DisruptionTriggers{
+						Inject: DisruptionTrigger{
+							NotBefore: metav1.NewTime(time.Now().Add(time.Minute * 5 * -1)),
+						},
+					}
+
+					Expect(newDisruption.ValidateCreate().Error()).Should(ContainSubstring("only values in the future are accepted"))
```

Comment:
nit you can also use the `MatchError` clause like that:

```suggestion
					Expect(newDisruption.ValidateCreate()).Should(MatchError(ContainSubstring("only values in the future are accepted")))
```